go/vt/vttablet/onlineddl/executor_test.go

```diff
@@ -120,6 +120,18 @@ func TestValidateAndEditAlterTableStatement(t *testing.T) {
 			alter:  "alter table t add fulltext key name0_ft (name0), add column i int, add fulltext key name1_ft (name1), add fulltext key name2_ft (name2)",
 			expect: []string{"alter table t add fulltext key name0_ft (name0), add column i int", "alter table t add fulltext key name1_ft (name1)", "alter table t add fulltext key name2_ft (name2)"},
 		},
+		{
+			alter:  "alter table t add constraint check (id != 1)",
+			expect: []string{"alter table t add constraint _aulpn7bjeortljhguy86phdn9 check (id != 1)"},
```

Comment:
We don't want to keep the default mysql name prefix here if it's short enough to fit? Or the naming as we do in the next test to stay consistent? Not a huge deal, but it starting with `_` seems a bit "off".

Reply:
done